src/components/TimeSlider.js

```diff
@@ -13,6 +13,9 @@ Date.prototype.toSliderString = function () {
   newDate[2] = year;
   return newDate.join('/');
 };
+Date.prototype.toJsonString = function () {
```

Comment:
it is not good idea to monkey patch the default window objects

 Could we use something like  as we already have date-fns in the project ?

```
import format from 'date-fns/format'

const getTodayDate = () => {
    const now = Date.now();

    return format(now, 'yyyy-MM-dd');
}
```

Reply:
let me change it

Reply:
It's done